docker-compose.yaml

```diff
@@ -1,26 +1,27 @@
 version: '3'
 
 services:
+
   peatio:
     build:
       context: peatio
       dockerfile: Dockerfile
+    restart: always
     depends_on:
       - db
+      - redis
+      - rabbitmq
     ports:
-      - "8000:80"
-    restart: always
+    - "80:8080"
     environment:
       RAILS_ENV: development
 
   db:
```

Comment:
I think we need to freeze mysql version and 5.7 might even be the correct version probably 5.6